advanced/Scripts/version.sh

```diff
@@ -136,8 +136,16 @@ errorOutput() {
 }
 
 defaultOutput() {
+    # Source the setupvars config file
+    # shellcheck disable=SC1091
+    . /etc/pihole/setupVars.conf
```

Comment:
`.` can be ambiguous and misinterpreted. 

```suggestion
     source /etc/pihole/setupVars.conf
```